backstop/scenarios/user-icon.js

```diff
@@ -1,3 +1,13 @@
 const { scenario } = require('../scenarioHelper');
 
 module.exports = [scenario('basic')];
+
+module.exports = [
+  // User icon types
+  scenario('Type default', {
+    selectors: ['#demo-default'],
+  }),
+  scenario('Type online status', {
+    selectors: ['#demo-status'],
+  }),
+];
```

Comment:
Forgot tests?

Reply:
Added tests but did not add hover tests yet.

Reply:
the animated one definitely needs hover, otherwise it's a pointless test

Reply:
Added test for animated hover on large size.

Reply:
Is the small size supposed to look like this?
![image](https://user-images.githubusercontent.com/9084735/122607222-404ddf00-d048-11eb-84ff-c3123a542698.png)

Reply:
Good catch.  I've fixed this by changing the stroke around the small user icons.  All other sizes have the same thickness stroke.  I tried scaling the stroke with the size of the user icon & it looked like too much on the larger sizes.  Here's what that would have looked like:
![Screen Shot 2021-06-22 at 12 08 16 PM](https://user-images.githubusercontent.com/849817/122961596-88277b80-d352-11eb-9391-1e10370a3246.png)